src/NLog.Web.AspNetCore/LayoutRenderers/AspNetResponseHttpsCompressionLayoutRenderer.cs

```diff
@@ -0,0 +1,51 @@
+﻿#if ASP_NET_CORE3
+using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Response HTTPS Compression Mode
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-response-https-compression}
+    /// </remarks>
+    [LayoutRenderer("aspnet-response-https-compression")]
+    public class AspNetResponseHttpsCompressionLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
+        /// Renders the Response HTTPS Compression Mode
```

Comment:
Just use `<inheritdoc/>` instead of duplicating the XML-docs. 

But please move the additional docs here into the class-docs.